### PR TITLE
Added changelog file and documentation for contributing. Updated README

### PR DESCRIPTION
**Pull request by chrisvanmook**
## Amsterdam styled components pull request

Before opening a pull request, please ensure:

- [ ] The default export from a file matches the file name
- [ ] The component styles are only placed in the `asc-core/src/components` folder. Exception is the `asc-ui/src/internals` folder where the styles are allowed. The components from this folder are just for internal use in the stories.
- [ ] The component style name follows the pattern `<ComponentName>Style/<ComponentName>Style.ts`
- [ ] Pull request has tests (we are going for 100% coverage!)

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+7. Eventually, we need to make our component available to use in other project, so we import
+   the corresponding component's default index.ts and make it available in the export.
+
+### Internals 
+`asc-ui/src/internals` folder where the styles are allowed. The components from this folder are 
+just for internal use in the stories.
+
+**Pro tip!** Use generators to quickly add the needed files to develop components.
+To make life a little easier, we added [hygen](https://www.hygen.io/). Simply run 
+`hygen component new`. You will be prompted with questions the generator needs to build the files
```
Comment (royderks) [inline docs/CONTRIBUTING.md]: Added a script in the `package.json` for this in https://github.com/Amsterdam/amsterdam-styled-components/pull/90

File: docs/MAINTAINERS.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Maintainers
+# Core Maintainers
 
 - chrisvanmook
```
Comment (royderks) [inline docs/MAINTAINERS.md]: Add link to github profile

File: README.md [hunk 3]
```diff
@@ -7,105 +7,98 @@
...
-- There is a known issue when developing with `yarn link` and using Hooks [click her for details](https://reactjs.org/warnings/invalid-hook-call-warning.html). The cause is that 2 React instances are used when using link. 
-- To solve this problem for the `./examples/create-react-app`, run
-```bash
-npm link ./examples/create-react-app/node-modules/react
-```
-from the root folder to use the same React version in the component lib as in the application.
-
-#### Generate components from templates
+3. Go to the repo where you want to use your package and run 
+   `yarn link <package-name>`.`<package-name>` can be found in the `package.json` you linked in
```
Comment (robinpiets) [inline README.md]: `yarn link <package-name>`. The`<package-name>`......
Add `The ` including a space to make it more readable

File: README.md [hunk 4]
```diff
@@ -7,105 +7,98 @@
...
-
-### Publishing
-
-To publish these packages to npm, do the following:
-
-1. Make sure you're logged in to npm, by running `npm login`
-2. Run `yarn publish:packages` to start lerna
-3. Select a version
+- There is a known issue when developing with `yarn link` and using Hooks 
+  [click her for details](https://reactjs.org/warnings/invalid-hook-call-warning.html). The cause
```
Comment (robinpiets) [inline README.md]: Typo

File: docs/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
+   build.
+2. Update the [CHANGELOG.md unreleased sections](../CHANGELOG.md#mardown-header-unreleased) with 
+   details of changes, this includes additions (new components or (theme) configurations), changes 
+   and removals.
+3. Once your PR is approved and merged, you can bump a version yourself or contact one of the 
+   [core maintainers](./MAINTAINERS.md) to discuss the version to bump. The versioning scheme we 
+   use is [SemVer](http://semver.org/).
+
+## Publishing
```
Comment (robinpiets) [inline docs/CONTRIBUTING.md]: Who can publish? Do you need write permissions?

**Pull request merged**